subprojects/docs/src/snippets/dependencyManagement/catalogs-settings/tests/resolve.sample.conf

```diff
@@ -1,2 +1,4 @@
 executable: gradle
 args: "dependencies --configuration compileClasspath -Dcreate4=true"
+# Do not fail for deprecation warnings: deprecated explicit creation of jmhRuntimeClasspath configuration
+flags: "--warning-mode=none"
```

Comment:
Disabling warnings outright seems non-ideal to me. What happens when jmh fixes this problem? This test would continue to ignore warnings and we wouldn't know. 

Seems like the problem is here: https://github.com/melix/jmh-gradle-plugin/blob/a278aeb4763a60ac78b334531f201304d3e0f61c/src/main/groovy/me/champeau/jmh/JMHPlugin.groovy#L320

Have we tried notifying the maintainers?